lib/internal/process/pre_execution.js

```diff
@@ -154,9 +154,16 @@ function setupUserModules(isLoaderWorker = false) {
   initializeESMLoader(isLoaderWorker);
   const CJSLoader = require('internal/modules/cjs/loader');
   assert(!CJSLoader.hasLoadedAnyUserCJSModule);
-  loadPreloadModules();
-  // Need to be done after --require setup.
-  initializeFrozenIntrinsics();
+  if (isLoaderWorker) {
+    // Loader workers are responsible for doing this themselves.
+    globalThis.loadPreloadModules = loadPreloadModules;
```

Comment:
Added globals must be documented in `globals.md`. But we should definitely NOT do that, adding globals would a breaking change.